### PR TITLE
Fix old configuration example in upgrading guide

### PR DESCRIPTION
**Pull request by ivanbrennan**
Fix a small mistake in the upgrading guide.

### CODE REVIEW

File: docs/UpgradeGuide.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Datadog.configure do |c|
   # Global settings
   c.tracer.hostname = '127.0.0.1'
   c.tracer.port = 8126
-  c.runtime_metrics.enabled = true
+  c.runtime_metrics_enabled = true
```
Comment (marcotc) [inline docs/UpgradeGuide.md:87]: 👋 @ivanbrennan, thanks for reaching out!

The UpgradeGuide is actually correct today.
You can see that `runtime_metrics.enabled` is indeed separated into two parts now:
https://github.com/DataDog/dd-trace-rb/blob/a349a7cd19c4881213365f5827b304c3518e800e/spec/datadog/core/configuration/settings_spec.rb#L506

Are you have any issues setting `c.runtime_metrics.enabled = ...` in your application?

Comment (ivanbrennan) [reply docs/UpgradeGuide.md:87]: The line I changed is supposed to show the old configuration. It's part of an example that shows an old config, followed by its equivalent new config.

Comment (marcotc) [reply docs/UpgradeGuide.md:87]: Ha, thank you for pointing that out!

**Review by marcotc — APPROVED** [review]
Thank you for catching this, @ivanbrennan! 🙇

**Pull request merged**